libs/bit_frames.c

```diff
@@ -587,7 +587,8 @@ static int extract_type2(struct extract_state* es)
 			struct switch_to_rel switch_to_rel;
 
 #ifdef DBG_EXTRACT_T2
-			fprintf(stderr, "#D %s:%i t2 gclk %i u16 0x%X\n",
+			fprintf(stderr,
+				"#D %s:%i t2 gclk %i u16 0x%"PRIX16"\n",
```

Comment:
you can (and should) make a space before and after the PRI macros to improve readability.